.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+version: 2
+jobs:
+  builder:
+    docker:
+      - image: docker:17.05.0-ce-git
+    resource_class: xlarge
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Login into Docker hub
+          command: docker login -u $DOCKER_USER -p $DOCKER_PASS
+      - run:
+          name: Build builder image
+          command: docker build -t titusoss/titus-executor-builder hack/builder
```

Comment:
`make push-builder` would be a bit better so we don't forget to change both places if we need to change how we build and push this image

Reply:
Since this runs in the "image: docker:17.05.0-ce-git" container, and that doesn't have make, we can't do that, unless we have another image based on that image that has the builder.

Reply:
gotcha. This is fine then.